FastSurferCNN/README.md

```diff
@@ -10,66 +10,77 @@ The network was trained with conformed images (UCHAR, 1-0.7 mm voxels and standa
+* `--lut`: FreeSurfer-style Color Lookup Table with labels to use in final prediction. Default: ./config/FastSurfer_ColorLUT.tsv
+* `--seg`: Name of intermediate DL-based segmentation file (similar to aparc+aseg).
+* `--cfg_cor`: Path to the coronal config file
+* `--cfg_sag`: Path to the axial config file
+* `--cfg_ax`: Path to the sagittal config file
 
 #### Checkpoints
-* --ckpt_sag: path to sagittal network checkpoint
-* --ckpt_cor: path to coronal network checkpoint
-* --ckpt_ax: path to axial network checkpoint
+* `--ckpt_sag`: path to sagittal network checkpoint
+* `--ckpt_cor`: path to coronal network checkpoint
+* `--ckpt_ax`: path to axial network checkpoint
 
 #### Optional commands
-* --clean: clean up segmentation after running it (optional)
-* --run_viewagg_on: Define where the view aggregation should be run on. 
+* `--clean`: clean up segmentation after running it (optional)
+* `--viewagg_device`: Define where the view aggregation should be run on. 
                     By default, the program checks if you have enough memory to run the view aggregation on the gpu. 
                     The total memory is considered for this decision. 
-                    If this fails, or you actively overwrote the check with setting "--run_viewagg_on cpu", view agg is run on the cpu. 
-                    Equivalently, if you define "--run_viewagg_on gpu", view agg will be run on the gpu (no memory check will be done).
-* --device <str>: Device for processing ("cpu","cuda") (optional, default: cuda)
-* --hires: Switch on high resolution processing (no conforming to 1mm, but to smallest voxel size).
-* --batch_size: Batch size for inference. Default=8
+                    If this fails, or you actively overwrote the check with setting `--viewagg_device cpu`, view agg is run on the cpu. 
+                    Equivalently, if you define `--viewagg_device gpu`, view agg will be run on the gpu (no memory check will be done).
+* `--device <str>`: Device for processing ("cpu","cuda") (optional, default: cuda)
+* `--hires`: Switch on high resolution processing (no conforming to 1mm, but to smallest voxel size).
+* `--batch_size`: Batch size for inference. Default=1
 
 
 ### Example Command Evaluation Single Subject
-To run the network on MRI-volumes of subjectX in ./data (specified by --t1 flag; e.g. ./data/subjectX/orig.mgz), change into the *FastSurferCNN* directory and run the following commands: 
+To run the network on MRI-volumes of subjectX in ./data (specified by `--t1` flag; e.g. ./data/subjectX/t1-weighted.nii.gz), change into the *FastSurferCNN* directory and run the following commands: 
 
 ```
-python3 run_prediction.py --t1 ../data/subjectX/orig.mgz \
---sd ../data \
+python3 run_prediction.py --t1 ../data/subjectX/t1-weighted.nii.gz \
+--sd ../output \
 --t subjectX \
---seg_log ../data/temp_Competitive.log \
+--seg_log ../output/temp_Competitive.log \
 --ckpt_sag ../checkpoints/aparc_vinn_sagittal_v2.0.0.pkl \
```

Comment:
But the example should probably not have the checkpoints any more. It should just use the defaults.

Reply:
This is resolved. See [here](https://github.com/Deep-MI/FastSurfer/pull/231)